src/ai/susi/server/api/cms/SkillMetricsDataService.java

```diff
@@ -188,6 +188,34 @@ public int compare(JSONObject a, JSONObject b) {
         JSONArray ratingsData = getSlicedArray(jsonValues, count);
         skillMetrics.put("rating", ratingsData);
 
+        // Get games skills based on ratings
+        Collections.sort(jsonValues, new Comparator<JSONObject>() {
+            @Override
+            public int compare(JSONObject a, JSONObject b) {
+                float valA;
+                float valB;
+                int result=0;
+
+                if (!a.get("group").toString().equals("Games, Trivia and Accessories")){
```

Comment:
I am a bit unhappy about the fixed group name here. This method does also not work with different kind of group categories like we would have with personas or private skills. I propose to use a variable which the client can set. Please add a post variable `topGroups` where a client can pass the string "Games, Trivia and Accessories".

That would make this feature more flexible.

Reply:
.. and use that variable instead of the fixed group name.